recipes/optuna-dashboard/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "optuna-dashboard" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
+  sha256: 277a81b0ebb33f405dfba2cd15d2ec628abd7b4acacd6364c88cea04ee3fa907
```

Hi @hadim. I've released the new version of optuna-dashboard. Could you update the version number and sha256 checksum?

```suggestion
{% set version = "0.4.0" %}


package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
  sha256: 29efd2e1df7e240d64546220a5086980073e2af25b62b2180ad305b5977ab116
```